### DISCUSSION TITLE
Is it possible to set a start value for Units like data_start_values?

### DISCUSSION THREAD

**Discussion by WohnungNDS** [open-sdg/open-sdg#1452] [Q&A] [answered]
Is it possible to set a start value for Units like data_start_values? 
Or do I have to change the data structure? [my data for 1-3-1](https://github.com/WohnungNDS/IM_Data/blob/develop/data/indicator_1-3-1.csv)

In my indicator [1.3.1](https://wohnungnds.github.io/IM_Site/1-3-1/) I have data with Units "in 1000" and "%". 
Opening the Indicator data  "%" schould be shown, but the radio button for "in 1000" is activated

![image](https://user-images.githubusercontent.com/67795376/136571777-4fccfda0-e7f7-4225-a145-770d2e6ee33b.png)

**Comment by LucyGwilliamAdmin** [comment, marked as answer]
@WohnungNDS from looking at [your data config file](https://github.com/WohnungNDS/IM_Data/blob/develop/config_data.yml#L125), you are using `sorting: default`, you could rearrange your data file as you say. This would change the order of them and then when the top one is selected it will be what you want.

But yes, you can use data_start_value - see this example: [metadata](https://github.com/ONSdigital/sdg-data/blob/develop/meta/3-5-1.md) and [indicator page](https://sdgdata.gov.uk/3-5-1/) - if you do this, they will stay in the same order but the second one will be selected

**Comment by WohnungNDS** [comment]
@LucyGwilliamAdmin Great! Thank you!

**Comment by WohnungNDS** [comment]
@LucyGwilliamAdmin 
Another question: Is it possible to have different data_start_values for different units?
in my case I have numbers (in 1000) and percent 

If percent is selected "Anteil der Personen mit MIgrationshintergrund im weiteren Sinne" is shown.
hitting "in 1000" I have to change th disaggregation manually to e.g. "Personen ohne Migrationshintergrund" before the data is shown.....

[indicator](https://wohnungnds.github.io/IM_Site/1-3-1/)

[metadata](https://github.com/WohnungNDS/IM_Data/edit/develop/meta/1-3-1.md)

**Comment by WohnungNDS** [comment]
@LucyGwilliamAdmin I think we can close this becaus of #1453

**Reply by LucyGwilliamAdmin** [reply]
As I've transferred it to a discussion, there's no way to close it like issues. Instead I have marked my response as an answer and locked the discussion
